lib/not_found_exception.rb

```diff
@@ -0,0 +1,11 @@
+class NotFoundException < ActionDispatch::PublicExceptions
+	def call(env)
+    	status = env["PATH_INFO"][1..-1]
+
+    	if status == "404"
+      		Rails.application.routes.call(env)
+    	else
+      		super
+    	end
+  	end
+end
```

Comment:
From my comment https://github.com/education/classroom/pull/510/files#r56814616 we don't need this file anymore